cfme/infrastructure/cluster.py

```diff
@@ -317,7 +317,10 @@ def step(self, *args, **kwargs):
         """Navigate to the correct view"""
         self.prerequisite_view.toolbar.view_selector.select('List View')
         version = self.obj.appliance.version
-        cluster_name = self.obj.short_name if version >= '5.8.1.2' else self.obj.name
+        if (version >= '5.7.4' and version < '5.8') or version >= '5.8.1.2':
```

Comment:
Gross. Not on your part though @izapolsk.  Hope we don't run into too many of these.

Reply:
I also hope so. but as you can see they merged some of 5.8 changes into 5.7.4